scout/adapter/mongo/variant_loader.py

```diff
@@ -695,24 +713,78 @@ def load_variants(
         else:
             rank_threshold = rank_threshold or 0
 
-        variants = vcf_obj(region)
-
         try:
-            nr_inserted = self._load_variants(
-                variants=variants,
-                variant_type=variant_type,
-                case_obj=case_obj,
-                individual_positions=individual_positions,
-                rank_threshold=rank_threshold,
-                institute_id=institute_id,
-                build=build,
-                rank_results_header=rank_results_header,
-                vep_header=vep_header,
-                category=category,
-                sample_info=sample_info,
-                custom_images=custom_images,
-                local_archive_info=local_archive_info,
-            )
+            if region == "":
```

Comment:
Its more simple but slightly less efficient, as the variant vcf obj has to be reset an extra time. But it can certainly be made more plain! I'll give it a shot!